jenkins.sh

```diff
@@ -52,7 +52,7 @@ if [[ ${GIT_BRANCH} != "origin/master" ]]; then
     app spec lib spec config
 fi
 
-GOVUK_CONTENT_SCHEMAS_PATH=tmp/govuk-content-schemas RAILS_ENV=test bundle exec rake
+GOVUK_CONTENT_SCHEMAS_PATH=tmp/govuk-content-schemas RAILS_ENV=test xvfb-run -a bundle exec rake
```

Comment:
It will just work. This is only a special flag for an error on jenkins when starting the webkit web driver.
I had faced a local error which was to do with asset routing, a fresh clone of the repo fixed it. It probably has something to do with some state change on my local repo after we changed the config to add an asset prefix.